geode-core/src/main/java/org/apache/geode/pdx/internal/PeerTypeRegistration.java

```diff
@@ -538,12 +540,10 @@ private void checkAllowed(boolean hasGatewaySender, boolean hasPersistentRegion)
     }
   }
 
-  /** Should be called holding the dlock */
-  private int getExistingIdForType(PdxType newType) {
+  private void buildTypeToIdFromRegion() {
```

Comment:
Since this builds both the typeToId and enumToId maps from the region it seems like you should name it so (like buildTypeAndEn umToIdMapsFromRegion).
We should be able to get rid of getExistingIdForEnum but need to verify the max stuff on with two different counters in this method.

Reply:
I think this would also be part of the enum ticket

Reply:
GEODE-7035